src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -2933,7 +2950,8 @@ address MacroAssembler::emit_trampoline_stub(int insts_call_instruction_offset,
 
   // make sure 4 byte aligned here, so that the destination address would be
   // 8 byte aligned after 3 intructions
-  while (offset() % wordSize == 0) { nop(); }
+  // RVC: when we reach here we may get a 2-byte alignment so need to align it
+  align(wordSize, NativeCallTrampolineStub::data_offset);
```

Comment:
Same question: Is there a reason to pass "NativeCallTrampolineStub::data_offset" as the "extra_offset" parameter here since this function will return (3 * NativeInstruction::instruction_size) which is multiple of 4 ?